lib/project-data-updater.js

```diff
@@ -88,8 +86,7 @@ ProjectDataUpdater.prototype.checkForAndImportUpdates = function(info) {
         .then(done, done);
     });
   } else {
-    this.done(new Error(this.fullName + ': no ' +
-      module.exports.ABOUT_YML + ' updates imported'));
+    this.done(true);
```

Comment:
Oh, and with `if (err === null)`, well need to call `done(null, true)` here. (Man, I do hate that boolean parameter, though.)